models/app.models.js

```diff
@@ -21,3 +21,26 @@ exports.selectArticles = (id) => {
             return res.rows;
         });
 }
+
+
+exports.updateArticle = (id, adjustedVotes) => {
+
+    let query = `SELECT votes FROM articles
+                    WHERE article_id = $1;`;
+                    
+    return db.query(query, [id])
+        .then(res => {
+
+            console.log('here');
```

Comment:
As before, remove a console.log as soon as you don't need it anymore to keep things nice and clean